Jenkinsfile_nightly

```diff
@@ -4,7 +4,7 @@ properties([
         pipelineTriggers([cron('00 20 * * *')]),
         parameters([
                 string(name: 'ENVIRONMENT', defaultValue: 'aat', description: 'Environment to test'),
-                string(name: 'URL_TO_TEST', defaultValue: 'https://probate.aat.platform.hmcts.net', description: 'The URL you want to run these tests against'),
+                string(name: 'URL_TO_TEST', defaultValue: 'https://probate.aat.platform.hmcts.net/caveats/start-apply', description: 'The URL you want to run these tests against'),
```

Comment:
will setting the URL here to a specific page break the functional tests? do the functional tests require a base URL to add specific page endings to, like in other repos?

Reply:
it won't break the functional tests . We have two shell scripts 1) run-functional-tests.sh  2)  run-nightly-functional-tests.sh
 run-nightly-functional-tests.sh --> it calls "test:nightly" script  to run e2e tests marked with tag "@nightly" so i don't see any issue there. Let's have a call?